### ISSUE TITLE
use debianbts with other Debbugs instances?

### ISSUE THREAD

**Issue by sandrotosi** [venthur/python-debianbts#46]
Hello,
do you know if it's possible to use `debianbts` to interact with other Debbugs instances (most notably https://debbugs.gnu.org/)?

i tried overwriting `debianbts.debianbts.URL` but that didnt change the actual SOAP endpoint used by `debianbts`.

having the ability to inspect bugs in other Debbugs instances will greatly help `bts-link`.

thanks,
Sandro

**Comment by venthur** [comment]
Hi Sandro,

in principle it should be possible. How did you try it so far? Importing, and changing the `URL` after importing? Check out the logic of `_build_soap_client` and how it is used, that should help.

Cheers,

Bastian

**Comment by gaetano-guerriero** [comment]
Try setting `debianbts._soap_client_kwargs.location = 'https://bugs.debian.org/cgi-bin/soap.cgi'`. 
If it works a cleaner interface to do this could be added.

**Comment by sandrotosi** [comment]
nice! that works indeed:

```
In [1]: from debianbts import debianbts

In [2]: debianbts._soap_client_kwargs['location'] = 'https://debbugs.gnu.org/cgi/soap.cgi'

In [3]: debianbts.get_status(18871)[0].subject
Out[3]: '24.4; nxml-mode: DTD edition breaks the file structure'
```

of course, a more "elegant" way to point to a different debbugs instance would be preferred, but that's already a good start for me

**Comment by venthur** [comment]
That feature is available since 3.2.0. Closing this issue.
